### PR TITLE
feat: Allow to type Unicode characters

### PR DESCRIPTION
**Pull request by mykola-mokhnach**
https://github.com/appium/appium/issues/13154

### CODE REVIEW

File: lib/tools/settings-client-commands.js [hunk 1]
```diff
@@ -442,4 +444,27 @@ commands.getSmsList = async function getSmsList (opts = {}) {
...
+/**
+ * Types the given Unicode string.
+ * It is expected that the focus is already put
+ * to the destination input field before this method is called.
+ *
+ * @param {string} text The string to type
+ * @returns {boolean} `true` if the input text has been successfully sent to adb
+ */
+commands.typeUnicode = async function typeUnicode (text) {
+  if (_.isNil(text)) {
```
Comment (imurchie) [inline lib/tools/settings-client-commands.js]: Why this check and not just having a default string of `''` and allowing the check below to return `false`?

Comment (mykola-mokhnach) [reply lib/tools/settings-client-commands.js]: I want all values except of null and undefined to be actually printed as they are (false -> `false`, `0` -> '0', etc). The `text` parameter will be received from the client via `mobile:` command

Comment (imurchie) [reply lib/tools/settings-client-commands.js]: Ok. Only `null` would work differently with a default parameter.

Comment (mykola-mokhnach) [reply lib/tools/settings-client-commands.js]: yep, I know. Thanks for the hint

**Pull request merged**